src/rules/at-rule-name-space-after/index.js

```diff
@@ -0,0 +1,67 @@
+import {
+  report,
+  ruleMessages,
+  validateOptions,
+  whitespaceChecker,
+} from "../../utils"
+
+export const ruleName = "at-rule-name-space-after"
+
+export const messages = ruleMessages(ruleName, {
+  expectedAfter: (name) => `Expected single space after \"${name}\"`,
+  rejectedAfter: (name) => `Unexpected whitespace after \"${name}\"`,
+})
+
+export default function (expectation) {
+  const checker = whitespaceChecker("space", expectation, messages)
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, {
+      actual: expectation,
+      possible: [
+        "always",
+        "never",
```

Comment:
How can expectation be "never"? That's kind of impossible, because you need at least one space to distinguish the at-rule name from its parameters. Right?